csi-rclone/README.md

```diff
@@ -0,0 +1,35 @@
+# CSI plugin for rclone mount
+
+This helm chart helps setting up resources for https://github.com/wunderio/csi-rclone storage plugin
+
+## Requirements
+
+Service account for setting up resources into kube-system namespace. 
+
+## Usage
+
+1. Set up storage backend. You can use [Minio](https://min.io/), Amazon S3 compatible cloud storage service.
+
+2. Either:
+ a. configure rclone defaults by pushing [rclone secret](https://github.com/wunderio/csi-rclone/blob/master/example/kubernetes/rclone-secret-example.yaml) to kube-system namespace and set `use-secret: true` in `values.yaml` 
```

Comment:
Instead of "pushing rclone secret to kube-system namespace", could we rather say "creating a secret in the kube-system namespace". "Pushing" is usually used in a git context.

Reply:
Fix amended into original commit. There was a slight change already, secret can be created in current namespace, it does not need `kube-system` anymore. This allows creating multiple csi-rclone instances in different namespaces with different credentials, having multiple storage options (even though PV already allows doing the same)